src/services/PhotoSearch.js

```diff
@@ -67,6 +70,23 @@ export default async function(onlyFavorites = false, options = {}) {
 			</d:eq>`
 		: ''
 
+	const onThisDay = options.onThisDay
+		? `<d:or>${Array(20).fill(1).map((_, years) => `<d:and>
+				<d:gt>
+					<d:prop>
+						<d:getlastmodified />
```

Comment:
the behavior will be weird, cause lastmodified is impacted when you tag an image for instance, so it won't be exactly "on this day"

but I guess that's the best we can do with Dav

Reply:
Once https://github.com/nextcloud/server/pull/30366 is merged this could be changed to creation_time

Reply:
ah interesting, I always thought this would cause performance issues

@skjnldsv am I wrong?